vignettes/aggregation.Rmd

```diff
@@ -14,24 +14,28 @@ COVID-19 cases in MA and VT.
 
 ```{r, message = FALSE}
 library(covidcast)
+library(delphi.epidata)
 library(epiprocess)
 library(dplyr)
 
 # Use covidcast::county_census to get the county and state names
-y <- covidcast::county_census %>% 
+y <- covidcast::county_census %>%
   filter(STNAME %in% c("Massachusetts", "Vermont"), STNAME != CTYNAME) %>%
   select(geo_value = FIPS, county_name = CTYNAME, state_name = STNAME)
 
 # Fetch only counties from VT and NH, then append names columns as well
-x <- covidcast_signal("jhu-csse", 
-                      "confirmed_incidence_num",
-                      start_day = "2020-06-01", 
-                      end_day = "2021-12-31",
-                      geo_type = "county", 
-                      geo_values = y$geo_value) %>% 
+x <- covidcast(
+  data_source = "jhu-csse",
+  signals = "confirmed_incidence_num",
+  geo_type = "county",
+  time_type = "day",
+  time_values = epirange(20200601, 20211231),
+  geo_values = paste(y$geo_value, collapse = ",")
```

Comment:
Very minor, but the order you specify these four arguments seems to be a bit inconsistent across different calls to `covidcast()` in different vignettes.

I would just choose one preset order for these four arguments (all common arguments that get called in `covidcast()` across different vignettes). 

I happen to like this order: `geo_type`, `time_type`, `geo_values`, `time_values`, since it kind of fits with the column ordering in an `epi_df` (and its metadata).

Reply:
There is also the possibility of using the current default order in the package 
```r
delphi.epidata::covidcast(data_source, signals, time_type, geo_type, time_values, geo_values, as_of = NULL, issues = NULL, lag = NULL)
```

No real preference as to the order, but agree with consistency.

Reply:
Sorry I'm so late to this. I didn't get any notifications that you were still working on this PR. 

Ok with either ordering as long as it's consistent. I see you've already done in a [7d95ad9](https://github.com/cmu-delphi/epiprocess/pull/59/commits/7d95ad939232c84ae9a849488b37ceec8f0fdf86) below. Thanks!